test/cql-pytest/conftest.py

```diff
@@ -19,6 +19,11 @@
 
 from util import unique_name, new_test_table, cql_session
 
+
+print(f"Driver name {DRIVER_NAME}")
+print(f"Driver version {DRIVER_VERSION}")
```

Comment:
Silly nitpick - why not a single line? Something like
print(f"Driver used: {DRIVER_NAME} {DRIVER_VERSION}")